### PR TITLE
Added a new method so you can search for a users devices

### PR DESCRIPTION
**Pull request by jontore**

### CODE REVIEW

File: src/entities/User.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ import { ajax } from '../tools/ajax.js';
 import Device from './Device';
 
 export default class User {
-    constructor(config) {
+    constructor(config, token) {
```
Comment (SmbatYeranyan) [inline src/entities/User.js]: Why is token used here? It should be passed down to the singleton Ajax beforehand

Comment (jontore) [inline src/entities/User.js]: It does not work if you want to use the custom ajax, that for instance points to dev.

Comment (jontore) [inline src/entities/User.js]: Can get ```token``` from the options in the ```ajax``` singelton

File: src/entities/User.js [hunk 2]
```diff
@@ -12,8 +13,10 @@ export default class User {
                 resolve(this.userInfo);
             } else {
                 ajax.get('/oauth2/user-info').then((response) => {
-                    this.userInfo = response;
-                    resolve(response);
+                    this.userInfo = Object.assign({}, response, {
```
Comment (SmbatYeranyan) [inline src/entities/User.js]: awesome!

**Comment by SmbatYeranyan** [comment]
Exactly what I meant! good to go 👍

**Pull request merged**